deltachat-jsonrpc/src/api/events.rs

```diff
@@ -305,6 +315,9 @@ impl From<EventType> for JSONRPCEventType {
                 chat_id: chat_id.to_u32(),
                 msg_id: msg_id.to_u32(),
             },
+            EventType::IncomingMsgBunch { msg_ids } => IncomingMsgBunch {
+                msg_ids: serde_json::to_string(&msg_ids).unwrap_or_default(),
+            },
```

Comment:
if you want to pass them over then do it asa ray of numbers (u32) and not as a json string, that workaround is only useful in the cffi bindings